COLLADAMaya/src/COLLADAMayaPhysXExporter.cpp

```diff
@@ -325,43 +295,46 @@ namespace COLLADAMaya
         //    return MS::kSuccess;
         //}
 
-        virtual MStatus onCompoundAttribute(MPlug & plug, const MString & name) override
+        virtual void onCompoundAttribute(MPlug & plug, const MString & name) override
         {
-            String nameStr(name.asChar());
-            mOpenCompoundAttributes.push(nameStr);
-            // openElement requires a String object as parameter that is not a temporary.
-            mPhysXExporter.getStreamWriter().openElement(mOpenCompoundAttributes.top());
-            return MS::kSuccess;
+            MObject object = plug.node();
+            MFnDependencyNode node(object);
+            MFnCompoundAttribute compoundAttribute(plug.attribute());
+            // Compound extra attributes are always Vector of 3 doubles
```

Comment:
You can only create double3 compound attributes in Maya. But after more reflection, this limitation comes from Maya UI only. This might change in the future and you also can create extra attributes programmatically with no limitation on attribute type. I'm going to fix that.